### DISCUSSION TITLE
Star Badge

### DISCUSSION THREAD

**Discussion by bryandbernhardt** [community/community#63531] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi, guys! I would like to earn the star badge on my repository, give me a like: https://github.com/bryandbernhardt/PI-VB-backend
Thanks =)

**Comment by matheus-foscarinid** [comment, marked as answer]
Hey there! Your repository looks fantastic, and I'd be more than happy to give it a well-deserved star! 🌟 Keep up the great work on your project! 💪

**Reply by bryandbernhardt** [reply]
thanks!

**Comment by akinwumidi** [comment]
@bryandbernhardt Your repository looks good, lots of efforts and work you have put in there, I left a star. it worth a million more stars

**Comment by DSDmark** [comment]
Hi @bryandbernhardt,

Here is tip how to could be able to get star:

- Add topics on your repo as much as you can (don't span, ok) which are related to your repo like if your repo project related to **ReactJS** then you can add **ReactJS**, **JavaScript**, **HTML** and **CSS** tags.

- Submit your project as collaboration project on [**GitHub education exchange community**](https://education.github.com/).

- Make your **READEME.md** file more readable like adding some **image** or **GIF** images

- Add a unique topic which is only related to your repo only like here is my unique topic for that repo **shoes-website** if you search that on Google you can see my repo on top like this: (shoes website GitHub project)

here is my repo (not totally good repo :face_with_spiral_eyes: ) you can see to make your [**my repo**](https://github.com/DSDmark/Shoes-webTemplate)
